graphql/resolvers/users.js

```diff
@@ -1089,5 +1099,25 @@ module.exports = {
       });
       return users;
     },
+
+    async insertPersEmailProp() {
+      var users = await User.find();
```

Comment:
I know we are just running this once, but the user model is one of our biggest (they have a bunch of props already). can you use the .select() option here to reduce the amount of data being transmitted

Reply:
Got rid of these two queries in exchange for updateMany(), which accomplishes the same thing and does not return any user records or unnecessary data.